test/integration/query-interface/schemas.test.ts

```diff
@@ -0,0 +1,38 @@
+import { expect } from 'chai';
+import { spy } from 'sinon';
+import type { CreateSchemaQueryOptions } from '@sequelize/core/_non-semver-use-at-your-own-risk_/dialects/abstract/query-generator';
+import { sequelize } from '../support';
+
+const queryInterface = sequelize.queryInterface;
+
+const testSchema = 'testSchema';
+
+if (sequelize.dialect.supports.schemas) {
+  describe('QueryInterface#createSchema', async () => {
+
```

Comment:
Can we avoid nesting by using `if (!sequelize.dialect.supports.schemas)` here and return directly?

Reply:
Done - amended the latest commit and force-pushed due to how small the change is @WikiRik